RELEASE.md

```diff
@@ -0,0 +1,3 @@
+Release type: patch
+
+This release adds support for GrahpQL-core 3.3
```

Comment:
shall we mention that 3.3 hasn't been released yet? (and that we support both versions?)